tests/tests_full_scan_engine_udf.py

```diff
@@ -332,9 +333,9 @@ async def test_udf_sqlite(self):
       # Run the query on the aidb database
       logger.info(f'Running query {aidb_query} in aidb database')
       aidb_res = aidb_engine.execute(aidb_query)
-      # TODO: may have problem with decimal number
       assert len(gt_res) == len(aidb_res)
       assert Counter(gt_res) == Counter(aidb_res)
+      assert sorted(gt_res) == sorted(aidb_res)
```

Comment:
'assert Counter(gt_res) == Counter(aidb_res)' is also used to check equality for all elements. You can remove this.

Reply:
Shouldn't we remove this?